ingestion_tools/scripts/importers/visualization_config.py

```diff
@@ -147,20 +195,22 @@ def get_annotation_layer_info(self, alignment_metadata_path: str) -> dict[str, A
                 color_seed = generate_hash({**annotation_hash_input, **{"shape": shape}})
                 hex_colors, float_colors = colors.get_hex_colors(1, exclude=colors_used, seed=color_seed)
 
-                path = self.config.to_formatted_path(
-                    os.path.join(precompute_path, f"{metadata_file_name}_{shape.lower()}"),
-                )
+                file_path, voxel_spacing, ratio = self._find_annotation_metadata(metadata_file_name, shape)
+
+                path = self.config.to_formatted_path(file_path)
 
                 is_instance_seg = shape == "InstanceSegmentation"
 
                 annotation_layer_info[file.get("path")] = {
                     "shape": shape,
+                    "voxel_spacing_ratio": ratio,
                     "args": {
                         "source_path": path,
                         "file_metadata": file,
                         "name_prefix": name_prefix,
                         "color": hex_colors[0],
                         "shape": shape,
+                        "resolution": (voxel_spacing * 1e-10,) * 3,
```

Comment:
This part is for the conversion from or to angstrom?

Reply:
Exactly yes! converting from angstrom to metres, one angstrom is 1e-10 meters